packages/uif/src/SliceIndexer.ts

```diff
@@ -0,0 +1,115 @@
+import { ChildIndexer } from './BaseIndexer'
+
+export type SliceHash = string
+
+export interface SliceState {
+  sliceHash: SliceHash
+  height: number
+}
+
+export interface SliceUpdate {
+  sliceHash: SliceHash
+  from: number
+  to: number
+}
+
+export abstract class SliceIndexer extends ChildIndexer {
+  override async update(from: number, to: number): Promise<number> {
+    const sliceStates = await this.getSliceStates()
+    const { toUpdate, toRemove } = diffSlices(
+      this.getExpectedSlices(),
+      sliceStates,
+      from,
+      to,
+    )
+
+    this.logger.info('Update', { toUpdate, toRemove })
```

Comment:
We shouldn't log the entire arrays

Reply:
Is it fine if we log it in `debug` mode? Or should we drop it entirely?